src/module.ts

```diff
@@ -368,6 +368,9 @@ export const plugin = new PanelPlugin<PanelOptions>(PlotlyPanel)
       FieldConfigProperty.NoValue,
       FieldConfigProperty.Decimals,
       FieldConfigProperty.Mappings,
+      FieldConfigProperty.Thresholds,
+      FieldConfigProperty.Unit,
+      FieldConfigProperty.DisplayName,
```

Comment:
Disabling some more standard options that either don't do anything at all or don't work as expected.